terraform/environments/prod/main.tf

```diff
@@ -0,0 +1,24 @@
+variable "db_password" {
+  description = "The password to use for TDS DB access"
+  type        = "string"
+}
+
+module "stack" {
+  source        = "../../template"
+
+  env_name = "prod"
+  db_id = "philip-testing"
+  instance_name_tag = "na-teddy"
+  db_password = "${var.db_password}"
+}
+
+terraform {
+  backend "s3" {
+    bucket     = "network-adequacy-terraform"
+    key        = "na-testing/terraform.tfstate"
+    region     = "us-west-1"
+
+    # ddb table to hold tfstate locks.
```

Comment:
How is that setup?

Reply:
_automagically_

Reply:
Yeah for real though, it's a feature of TF that requires creating the ddb table manually.

Reply:
(You can't create the state lock table using this terraform state because it won't be able to get the state lock to create the table with lol)